### DISCUSSION TITLE
Installation with Homebrew

### DISCUSSION THREAD

**Discussion by henioStraszny** [grafviktor/goto#53] [Ideas]
I'd like to bring in the feature, to install GOTO with Homebrew, I think that this is very popular way of getting software, and should be one of the simplest. 

Please let me know if that this ok to start my work on it.

**Comment by grafviktor** [comment]
Hello Jacek. At this moment, I can only build `deb` and `rpm` packages. That would be really great if you could integrate with homebrew!
